Directory.Packages.props

```diff
@@ -19,7 +19,7 @@
     <PackageVersion Include="System.ValueTuple" Version="4.5.0" />
   </ItemGroup>
 
-  <ItemGroup>
+  <ItemGroup Condition="'$(DotNetBuildFromSource)' != 'true'">
     <GlobalPackageReference Include="Microsoft.DotNet.ApiCompat.Task" Version="9.0.100-preview.2.24102.10" />
```

Comment:
There's a git merge conflict with this line. In `main`, the version is `9.0.100-rc.2.24474.11`.